jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/assembly/AssemblyManagerVerifyAssemblyReferencedInDockerfileTest.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.build.api.assembly;
+
+import org.eclipse.jkube.kit.common.AssemblyConfiguration;
+import org.eclipse.jkube.kit.common.KitLogger;
+import org.eclipse.jkube.kit.config.image.build.BuildConfiguration;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Properties;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.mock;
+
+@RunWith(Parameterized.class)
+public class AssemblyManagerVerifyAssemblyReferencedInDockerfileTest {
+
+    KitLogger logger;
+    @Before
+    public void setUp(){
+        logger=spy(KitLogger.SilentLogger.class);
+    }
+    @Parameterized.Parameters(name = "{0}: verifyDockerFile({1})={2}")
```

Comment:
The test description(s) seem(s) to be updated, and is better now.

However, we're still not mentioning what we're actually testing.

I assume we're testing that the verify method logs n warnings (or none) provided different Dockerfile(s).
It would be good if we could make this clearer.

Reply:
Done!